aya-obj/README.md

```diff
@@ -0,0 +1,41 @@
+# aya-obj - an eBPF object file loading library
+
```

Comment:
I've added a new section both in the README and in rustdoc. Is something like this OK?
```markdown
## Status

NOTE: this crate includes code that started as internal API used by
the [aya] crate. It has been split out so that it can be used by
other projects that deal with eBPF object files. Unless you're writing
low level eBPF plumbing tools, you should not need to use this crate
but see the [aya] crate instead.

The API as it is today has a few rough edges and is generally not as
polished nor stable as the main [aya] crate API. As always,
improvements welcome!

[aya]: https://github.com/aya-rs/aya
```

Reply:
Perfect!